jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/MavenScmEnricherTest.java

```diff
@@ -106,29 +103,7 @@ public void testMavenScmOnlyDevConnection() {
     }
 
     @Test
-    public void testMavenScmOnlyUrl() {
```

Comment:
this test seems to be missing

Reply:
This test is same as https://github.com/eclipse/jkube/blob/22a2c40b8599cadbdac79be6d49e2069930528b4/jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/MavenScmEnricherTest.java#L65 so I removed it.

Reply:
Because both tests are asserting exactly same thing. They differ only in their name.

`testMavenScmOnlyConnection `
https://github.com/eclipse/jkube/blob/22a2c40b8599cadbdac79be6d49e2069930528b4/jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/MavenScmEnricherTest.java#L80-L83

`testMavenScmOnlyUrl `
https://github.com/eclipse/jkube/blob/22a2c40b8599cadbdac79be6d49e2069930528b4/jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/MavenScmEnricherTest.java#L124-L127

Reply:
> What I'm asking is if you randomly deleted a test.

No, because of this I removed `testMavenScmOnlyUrl` test
 https://github.com/eclipse/jkube/blob/22a2c40b8599cadbdac79be6d49e2069930528b4/jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/MavenScmEnricher.java#L40

Anyways fixed it. 
Really sorry for misunderstanding 😔